include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -318,8 +331,26 @@ ORT_API_STATUS(OrtGetStringTensorDataLength, _In_ const OrtValue* value, _Out_ s
 ORT_API_STATUS(OrtGetStringTensorContent, _In_ const OrtValue* value, _Out_ void* s, size_t s_len,
                _Out_ size_t* offsets, size_t offsets_len);
 
-ORT_API_STATUS(OrtTensorProtoToOrtValue, _Inout_ OrtAllocator* allocator,
-               _In_ const void* input, int input_len, _Out_ OrtValue** out);
+/**
+ *
+ * @param input
+ * @param input_len
+ * @param input_file_path A local file path of where the input was loaded from. Can be NULL
```

Comment:
>A local file path [](start = 26, length = 17)

Can we be explicitly about whether the path could/should be relative or absolute? If relative, relative to what? 
Would also be good to say when it should be null.